mypy/checker.py

```diff
@@ -2357,6 +2357,10 @@ def visit_raise_stmt(self, s: RaiseStmt) -> None:
     def type_check_raise(self, e: Expression, s: RaiseStmt,
                          optional: bool = False) -> None:
         typ = self.expr_checker.accept(e)
+        if isinstance(typ, TypeType):
```

Comment:
Shouldn't (for consistency) the same case be added to `check_except_handler_test` (with test cases)?